h3_test.go

```diff
@@ -231,6 +231,22 @@ func TestUncompactCells(t *testing.T) {
 	assertCellIn(t, validCell, out)
 }
 
+func TestChildPosToCell(t *testing.T) {
+	t.Parallel()
+
+	childrens := validCell.Children(6)
+
+	assertEqual(t, childrens[0], validCell.ChildPosToCell(0, 6))
+}
+
+func TestChildPos(t *testing.T) {
+	t.Parallel()
+
+	childrens := validCell.Children(7)
+
+	assertEqual(t, 32, childrens[32].ChildPos(5))
```

Comment:
because the significance of 5 is that it is the validCell's resolution, can you use `validCell.Resolution()`?

Reply:
makes total sense